third_party/ucd/emoji-data.txt

```diff
@@ -0,0 +1,1315 @@
+# Emoji Data for UTR #51
```

Comment:
Since this is coming from another location than the normal UCD data files, you also need to updated the README file in the ucd directory to point to its location and version.